recipe/meta.yaml

```
@@ -14,14 +14,15 @@ source:
 build:
   skip: true  # [win]
   script: {{ PYTHON }} -m pip install . -vv
-  number: 1
+  number: 2
 
 requirements:
   build:
     - cython                                 # [build_platform != target_platform]
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
     - numpy                                  # [build_platform != target_platform]
+    - scipy                                  # [build_platform != target_platform]
```

I _think_ this needs to be the same as in the `requirements:host` list:

```suggestion
    - scipy <1.13.0                          # [build_platform != target_platform]
```